app/models/email/requests/SendEmailRequest.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.email.requests
+
+import cats.data.*
+import cats.implicits.*
+import models.BusinessType.SoleTrader
+import models.{BusinessType, IndividualName, SoleTraderName, UserAnswers, Utr}
+import pages.*
+import play.api.libs.json.*
+import queries.Query
+
+case class SendEmailRequest(to: List[String],
+                            templateId: String,
+                            parameters: Map[String, String])
+
+object SendEmailRequest {
+  implicit val format: OFormat[SendEmailRequest] = Json.format[SendEmailRequest]
+
+  private val RegistrationSubmittedTemplateId: String = "dprs_registration_submitted"
+
+  def apply(email: String, dprsId: String, name: String): SendEmailRequest = SendEmailRequest(
+    to = List(email),
```

Comment:
Nick mentioned that we should be sending this email to the secondary contact too if we have collected one

Reply:
I asked Kerry and she said we only need to send to the Primary contact. I can double check with her again.